src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -67,4 +76,13 @@ public boolean equals(Object other) {
         return id.getText().equals(card.id.getText())
                 && person.equals(card.person);
     }
+
+    /**
+     * Returns the color style for {@code tagName}'s label.
+     */
+    private String getTagColorStyleFor(String tagName) {
+        // use hash code of tag name to generate random color, so color remains consistent between different runs
+        return TAG_COLOR_STYLES[Math.abs(tagName.hashCode()) % TAG_COLOR_STYLES.length];
```

Comment:
I feel that we should explicitly note that Java API dictates String hashing (specifically) to be consistent across different runs.

Reply:
Edit comment to note this down